examples/cull-idle/Cull_service_UI/web_service.py

```diff
@@ -0,0 +1,132 @@
+"""
+Task :  Web service for Cull UI
+
+Date : 16 Aug 2016
+
+Version : 1.0
+
+Author : Vigneshwer
+"""
+
+# Loading python modules 
+import tornado.ioloop
+import tornado.web
+import tornado.escape
+from tornado.concurrent import Future
+from tornado.gen import coroutine
+from tornado.options import define, options, parse_command_line
+from tornado.log import app_log
+from tornado.httpclient import AsyncHTTPClient, HTTPRequest
+import tornado.websocket
+import config
+import datetime
+import json
+import os
+import datetime
+from dateutil.parser import parse as parse_date 
+import json
+
+define("port", default=8888, help="run on the given port", type=int)
+define("debug", default=False, help="run in debug mode")
+define("static_path", default="./", help="Content of the template")
+define('url', default='https://127.0.0.1:443/hub', help="The JupyterHub API URL")
+
+# Values to be changed
+api_token="c4297f02c25b49c2a63c9b102b24cc51"
```

Comment:
it's a good idea to get the API token from an environment variable. See the existing cull_idle example for reference. The same goes for the other constants below - they can come from the options above, and don't need to be defined as constants here.